features/api/library_creation_requests.feature

```diff
@@ -19,7 +19,7 @@ And I have a "full" authorised user with the key "cucumber"
 
     Given I have an active study called "Testing the sequencing requests API"
     And the UUID for the study "Testing the sequencing requests API" is "11111111-2222-3333-4444-000000000000"
-    And I have a library tube of stuff called "tube_1"
```

Comment:
This step actually creates an empty library tube. There was a todo above it, written ten years ago, saying perhaps it should be a full tube. I've updated the step text to reflect what it actually does, and removed the todo.